Microsoft.Toolkit.Uwp.UI.Controls/RadialProgressBar/RadialProgressBar.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    
+    <Style TargetType="local:RadialProgressBar" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:RadialProgressBar">
+                    <Grid>
+                        <!-- Gray outline of progress bar -->
+                        <Path Fill="Transparent" Stroke="{TemplateBinding Background}" StrokeThickness="{TemplateBinding Thickness}" StrokeDashCap="Flat">
```

Comment:
Typically you wouldn't use "Background" as the track background here, you'd create a separate DependencyProperty for it. As odd as it may be, Background should be template bound to the grid above this, and set as Transparent by default imo.